### ISSUE TITLE
[Bug] CodeSnippet is missing focus state in Safari

### ISSUE THREAD

**Issue by thyhmdo** [carbon-design-system/carbon#20473]
### Issue
Keyboard interaction is missing in Safari
When you start tabbing into the [Code snippet,](https://react.carbondesignsystem.com/?path=/story/components-codesnippet--multiline) it either takes focus on the content or the browser. 

### Suggestion
Check the Keyboard interaction guide [here](https://carbondesignsystem.com/components/code-snippet/accessibility/) and add that to the Storybook

**Comment by tay1orjones** [comment]
In chrome this is working as expected - were you testing in firefox? I can't find a [corresponding bugzilla report](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=textbox+focus), so I'm not sure what's going on with this one.
